lib/src/doc/lives.rs

```diff
@@ -30,26 +36,66 @@ impl<'a> Document<'a> {
 			for lv in self.lv(opt, txn).await?.iter() {
 				// Create a new statement
 				let lq = Statement::from(lv);
-				// Check LIVE SELECT where condition
-				if let Some(cond) = lq.conds() {
-					// Check if this is a delete statement
-					let doc = match stm.is_delete() {
-						true => &self.initial,
-						false => &self.current,
-					};
-					// Check if the expression is truthy
-					if !cond.compute(ctx, opt, txn, Some(doc)).await?.is_truthy() {
-						continue;
-					}
+				// Get the event action
+				let met = if stm.is_delete() {
```

Comment:
Maybe this could be the Notification Activity From<Statement>. It doesn't do that yet, but wondering if it would be convenient

Reply:
Yeah we could do this 👍 .